docs/tutorials/go/build-apps/get-started.md

```diff
@@ -0,0 +1,176 @@
+
+1. Via `websocket`:
+    ```go
+	// Define the network client
+	client := websocket.NewClient(websocket.NewClientConfig().
+		WithHost("wss://s.altnet.rippletest.net:51233"))
+    
+	// Disconnect the client when done. (Defer executes at the end of the function)
+	defer client.Disconnect()
+
+	// Connect to the network
+	if err := client.Connect(); err != nil {
+		fmt.Println(err)
+		return
+	}
+    ```
+2. Via `RPC`: 
+    ```go
+    cfg, err := rpc.NewClientConfig("https://s.altnet.rippletest.net:51234/")
+    if err != nil {
+        panic(err)
+      }
+
+    // Initiate the network client
+    client := rpc.NewClient(cfg)
+    ```
+
+
+#### Connect to the production XRP Ledger
+
+The sample code in the previous section shows you how to connect to the Testnet, which is a [parallel network](../../../concepts/networks-and-servers/parallel-networks.md) for testing where the money has no real value. When you're ready to integrate with the production XRP Ledger, you'll need to connect to the Mainnet. You can do that in two ways:
+
+- By [installing the core server](../../../infrastructure/installation/index.md) (`rippled`) and running a node yourself. The core server connects to the Mainnet by default, but you can [change the configuration to use Testnet or Devnet](../../../infrastructure/configuration/connect-your-rippled-to-the-xrp-test-net.md). [There are good reasons to run your own core server](../../../concepts/networks-and-servers/index.md#reasons-to-run-your-own-server). If you run your own server, you can connect to it like so:
+
+  ```go
+  import 	"github.com/Peersyst/xrpl-go/xrpl/websocket"
+  const MyServer := "ws://localhost:6006/"
+  client := websocket.NewClient(websocket.NewClientConfig().WithHost(MyServer))
+  ```
+
+  See the example [core server config file](https://github.com/XRPLF/rippled/blob/c0a0b79d2d483b318ce1d82e526bd53df83a4a2c/cfg/rippled-example.cfg#L1562) for more information about default values.
+
+- By using one of the available [public servers][]:
+
+  ```go
+  import 	"github.com/Peersyst/xrpl-go/xrpl/websocket"
+  const PublicServer = "wss://xrplcluster.com/"
+  client := websocket.NewClient(websocket.NewClientConfig().WithHost(PublicServer))
+  ```
+
+### 2. Get account
+
+In `xrpl-go`, account creation and key management live in the wallet package, and on Testnet you can use the built-in faucet provider on your WebSocket (or RPC) client to fund a brand-new account immediately.
+
+Here we spin up a Testnet‐connected WebSocket client, generate a fresh ED25519 wallet, then fund it automatically via the public faucet.
+
+```go
+client := websocket.NewClient(
+	websocket.NewClientConfig().
+		WithHost("wss://s.altnet.rippletest.net:51233").
+		WithFaucetProvider(faucet.NewTestnetFaucetProvider()),
+)
+
+defer client.Disconnect()
+if err := client.Connect(); err != nil {
+	fmt.Println(err)
+	return
+}
```

Comment:
We've already shown the user how to connect with the client(s) in the previous step, so we probably don't need to show this again.